UX-Guide-Metadata/techniques/epub-metadata.html

```diff
@@ -268,26 +268,28 @@ <h5>UI</h5>
 					</section>
 				</section>
 			</section>
-			<section id="accessibility-conformance">
-				<h3>Accessibility Conformance</h3>
+
+
+            <section id="accessibility-conformance-links">
+				<h3>Accessibility Conformance (Links)</h3>
 				<p><b>Value</b>: Textual Link based on Metadata or raw URL found in EPUB Package Document</p>
 				<p>This technique relates to <a href="../principles/#accessibility-conformance">Accessibility Conformance principle</a>.</p>
 					
-				<aside class="example" id="cb10">
+				<aside class="example" id="cp-link">
 					<pre><code>&lt;link rel="dcterms:conformsTo"&gt;</code></pre>
 				</aside>
 				<div class="note">
 					<p>Interpret this metadata to be as human friendly as possible. So if the URI is:</p>
```

Comment:
Slightly tangential, but I see URI here and URL below. Probably should stick with URL everywhere.